### DISCUSSION TITLE
Funds doesnt show in Wallet

### DISCUSSION THREAD

**Discussion by svenna1980** [Chia-Network/chia-blockchain#7122] [Support]
Hello everyone,

a friend has send some chia to my public adress (3 days ago). Until now the transferd chia doesnt showed in my wallet.  I searched the public adress and found out, that the chia was succesful wrote to the blockchain.  I let the wallet sync again from zero in hope that than the funds were showed in my wallet. But this doesn´t solve the issue. 
Can someone help me?

best regards
Sven

**Comment by loppefaaret** [comment]
Hey Sven
i would love to help you out, but i need some additional information to do that - such as was the wallet fully synced, and not showing the transaction incoming ? perhaps you could provide the transaction ID, or the receive address that was used ?

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
